.github/templates/ios_ci_workflow.yml.j2

```
@@ -44,7 +44,7 @@ jobs:
   build:
     # NOTE: These builds will not run successfully without running on `pytorch/pytorch` due to the limitations
     #       of accessing secrets from forked pull requests and IOS' dependency on secrets for their build/test
-    if: ${{ github.event_name == 'push' || github.event.pull_request.head.repo.full_name == github.repository }}
+    if: ${{ github.repository_owner == 'pytorch' && (github.event_name == 'push' || github.event.pull_request.head.repo.full_name == github.repository) }}
```

Confirmed with @seemethere  this can just be 
```suggestion
    if: ${{ github.repository_owner == 'pytorch' }}
```